tasks.py

```diff
@@ -0,0 +1,49 @@
+from invoke import task
+import os
+
+
+@task
+def install_dependencies(c):
+    c.run("pip install -r requirements.txt")
+
+
+@task
+def check_format(c):
+    c.run("black --check python/*.py")
```

Comment:
Let's check the `app` dir as well - other settings come from the `pyproject.toml` file.
```suggestion
    c.run("black --check .")
```